test/jdk/javax/swing/JTextArea/8149849/DNDTextToScaledArea.java

```diff
@@ -51,17 +56,29 @@
     private static volatile boolean passed = false;
 
     public static void main(String[] args) throws Exception {
+        var lge = GraphicsEnvironment.getLocalGraphicsEnvironment();
+        for (GraphicsDevice device : lge.getScreenDevices()) {
+            test(device);
+        }
+    }
+
+    private static void test(GraphicsDevice device) throws Exception {
         Robot robot = new Robot();
-        robot.setAutoDelay(50);
+        robot.setAutoDelay(150);
 
-        SwingUtilities.invokeAndWait(DNDTextToScaledArea::createAndShowGUI);
+        SwingUtilities.invokeAndWait(() -> createAndShowGUI(device));
         robot.waitForIdle();
 
         SwingUtilities.invokeAndWait(() -> {
             srcPoint = getPoint(srcTextArea, 0.1);
             dstPoint = getPoint(dstTextArea, 0.75);
         });
         robot.waitForIdle();
+        // check the destination
+        robot.mouseMove(dstPoint.x, dstPoint.y);
+        robot.mousePress(InputEvent.BUTTON1_MASK);
+        robot.mouseRelease(InputEvent.BUTTON1_MASK);
+        robot.waitForIdle();
```

Comment:
Why is clicking the `dstTextArea` required?

Reply:
Mostly to make window active, when I stress testing the fix, this test rarely fails(before and after the fix) I think due to the window appeared inactive and DnD did not start/not complete.